templates/default/systemd/sensu-api.service.erb

```diff
@@ -0,0 +1,14 @@
+[Unit]
+Description=sensu api
+
+[Service]
+User=sensu
+Group=sensu
+EnvironmentFile=/etc/default/sensu
+ExecStart=/opt/sensu/bin/sensu-api -c /etc/sensu/config.json -d /etc/sensu/conf.d -e /etc/sensu/extensions -l /var/log/sensu/sensu-api.log
```

Comment:
Or should i now be able to  dump these completely and use the packages systemd files instead?